src/views/List.jsx

```diff
@@ -1,6 +1,13 @@
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	console.log(data);
+
+	// Q: const or let?
+	const listItem = data.map((item) => {
```

Comment:
this utility function works well! since you're not exporting it from this file (for something else to use) and you're only using it once, you may consider in the future just making a function like this an inline call, e.g.:
```jsx
<>
  <ul>
    {data.map(({id, name}) => {
      return <ListItem key={id} name={name} />;
    })}
  </ul>
</>
```

Reply:
and if you are planning to use `listItem` somewhere else, consider making the name more descriptive (ex. `listItem` as a single value vs `listItem` as an array of items)

Reply:
Would this code technically be less efficient than yours Matthew? In the return of the `List` component:

`	<ul>
				{data.map((item) => {
					return <ListItem key={item.id} name={item.name} />;
				})}
	</ul>`

Reply:
@alex-andria I've seen this done both ways, so I think it's more of a personal preference. In the React docs [here](https://reactjs.org/docs/lists-and-keys.html), they map over an array both outside of the return of the component and inline.

Reply:
@alex-andria if you were asking about whether or not destructuring changes efficiency in the function expression, I don't think it matters very much here. Same as DJ mentioned probably a personal preference, or in some cases a team may have a code style guide that favors one over the other. That said there are probably blogs that talk about destructuring, at length 😁